pkg/config/loader.go

```diff
@@ -59,7 +59,7 @@ func LoadConfig() (*Config, error) {
 
 	// 3 - figure out what address we will tell pods to dial for aggregation
 	if cfg.Aggregation.AdvertiseAddress == "" {
-		if ip, ok := os.LookupEnv("SONOBUOY_ADVERTISE_IP"); ok {
+		if ip := os.Getenv("SONOBUOY_ADVERTISE_IP"); ip != "" {
```

Comment:
This is because the `SONOBUOY_ADVERTISE_IP` env (as well as all expected environment variables) will always be passed from the [`run_master.sh`](https://github.com/heptio/sonobuoy/pull/20/files#diff-9160fb798e31c8d887cdb95a8f0051a1R19) script into the `sonobuoy` binary. Therefore `LookupEnv` will always return `true` for `ok` since we are always setting it, even if it's empty.

If there is a case of setting `SONOBUOY_ADVERTISE_IP` to `''` where we don't want the `else` block to execute then I'll have to rethink this strategy.